### ISSUE TITLE
Кэш установленных модулей.

### ISSUE THREAD

**Issue by Khorevaa** [oscript-library/opm#42]
Проблема:
Мне для проверки работы сделанного PR понадобилась свежая библиотека 1testrunner. 
Чтобы ее установить только локально - надо удалить из глобальной папки, при этом преключение на другой проект приводит к обратной операции. (или аналогичной) 

Посмотрел код и сделал вывод: кэш установленных библиотек работает только для "глобально" установленных. Что не очень корректно.

Как следствие, вызов команды opm install -l , в папке проекта, каждый раз устанавливает одни и те же библиотеки - локально, хотя они уже там есть.

Предлагаю переделать(доделать) флаг  -g, а при вызове opm install -l , в папке проекта,  устанавливать зависимости по умолчанию в локальную папку. 

Изменить(доделать) логику поиска и подбора закэшированных библиотек.(думаю вопрос к движку) По следующим приоритетам:
1. Поиск в локальной папке
2. Поиск в глобальной папке. 

Соответсвенно и загружать в работу их в том же порядке, сначала локальные библиотеки, потом глобальные. (и не грузить глобальную библиотеку, если загружена локальная той же версии)

**Comment by nixel2007** [comment]
да, это мой тех долг, я про него помню. требуется доработка кэша библиотек.
а загрузка локальных уже работает - надо доопределить lib.additional в oscript.cfg

**Comment by Khorevaa** [comment]
В принципе была идея написать самому. Если на ближайшей недели дойдут руки тогда напишу класс ЛокальныйКэш

**Comment by nixel2007** [comment]
Отдельный класс не нужен. Нужно просто дописать в место поиска содержимое lib.additional
